### PR TITLE
metainfo added

### LINKED ISSUE #17: Add favicon to gh-pg
30 Days of PWA blog series GitHub page needs a favicon

### PR DESCRIPTION
**Pull request by webmaxru**
close #17

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -2,11 +2,39 @@
...
 <head>
   <meta charset="UTF-8">
-  <title>Document</title>
+  <title>30 Days of PWA - Learning Series about Progressive Web Apps</title>
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-  <meta name="description" content="Learning content for Windows Student Developers">
+  <meta name="description" content="For 30 days, we publish articles that aim to introduce developers to Progressive Web App. We have content that covers 0-level to 200-level topics. Each post takes 5-10 minutes to read and is followed by a sample snippet or exercise.">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
 
+  <link rel="icon" href="https://microsoft.github.io/win-student-devs/30DaysOfPWA/_media/favicon.png" />
```
Comment (beth-panx) [inline docs/index.html:10]:
```suggestion
<link rel="icon" href="/30DaysOfPWA/_media/favicon.png" />
```

File: docs/index.html [hunk 2]
```diff
@@ -2,11 +2,39 @@
...
+  <meta property="og:type" content="website" />
+  <meta
+    property="og:title"
+    content="30 Days of PWA - Learning Series about Progressive Web Apps"
+  />
+  <meta
+    property="og:description"
+    content="For 30 days, we publish articles that aim to introduce developers to Progressive Web App. We have content that covers 0-level to 200-level topics. Each post takes 5-10 minutes to read and is followed by a sample snippet or exercise."
+  />
+  <meta property="og:image" content="https://microsoft.github.io/win-student-devs/30DaysOfPWA/_media/social.jpg" />
```
Comment (beth-panx) [inline docs/index.html:23]: Is there a size requirement for this image? `social.jpg` is the same as `kickoff.jpg`.

File: docs/index.html [hunk 3]
```diff
@@ -2,11 +2,39 @@
...
+  <meta property="twitter:url" content="https://microsoft.github.io/win-student-devs/#/30DaysOfPWA/" />
+  <meta
+    name="twitter:title"
+    content="30 Days of PWA - Learning Series about Progressive Web Apps"
+  />
+  <meta
+    name="twitter:description"
+    content="For 30 days, we publish articles that aim to introduce developers to Progressive Web App. We have content that covers 0-level to 200-level topics. Each post takes 5-10 minutes to read and is followed by a sample snippet or exercise."
+  />
+  <meta name="twitter:image" content="https://microsoft.github.io/win-student-devs/30DaysOfPWA/_media/social.jpg" />
```
Comment (beth-panx) [inline docs/index.html:36]: Same here. Duplicate image. Twitter banner size is `1024x512`. You can also use that file in the `./30DaysOfPWA/_media/1024x512` folder

**Review by beth-panx — CHANGES_REQUESTED** [review]

**Comment by beth-panx** [comment]
The social image is a duplicate of kickoff.jpg

File: docs/index.html (hunk 2 above)
Comment (webmaxru) [reply docs/index.html:23]: It's not about size but about ratio. This `kickoff.jpg` is too stretched horizontally so the left and right parts (incl. photos) are cut off on the preview. I just added some whitespace on top and bottom

File: docs/index.html (hunk 3 above)
Comment (webmaxru) [reply docs/index.html:36]: Image in this folder is different - it's contains only title. I want to use the graphics we have on `kickoff.jpg` for social banner

**Review by beth-panx — APPROVED** [review]

**Review by zateutsch — APPROVED** [review]

**Pull request merged**